.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+
+node_js:
+  - "4.1"
+  - "4.0"
+  - "0.12"
+  - "0.11"
+  - "0.10"
```

Comment:
I would really not test against 0.10 as it does not support a lot of really important ES6 features.